### ISSUE TITLE
Cannot specify location of RG "AzSKRG".

### ISSUE THREAD

**Issue by cmenzi** [azsk/DevOpsKit#193]
## When executing `Set-AzSKSubscriptionSecurity ... -TargetResourceGroup azsk` it still provisio a `AzSKRG` in "East US ".

Is it possible to specify that all resources gets provisioned to "West Europe"?

## Steps to reproduce
```
Set-AzSKSubscriptionSecurity -SubscriptionId xxx -SecurityContactEmails "xxx" -SecurityPhoneNumber "xxx" -TargetResourceGroup xxx -AlertResourceGroupLocation "West Europe"
```
## Expected behavior

It should be possible to specify to target location of all provisioned resources.

## Actual behavior

It always creates resources in "East US 2".

**Comment by VishalHaibatpure** [comment]
You can achieve resource group name and location customization using Org policy feature of AzSK.
Refer [FAQ](https://github.com/azsk/DevOpsKit-docs/tree/master/07-Customizing-AzSK-for-your-Org#is-it-possible-to-control-default-resource-group-name-azskrg-and-location-eastus2-created-for-azsk-components)

If you are not targeting for custom Org policy, you can create AzSKRG using Azure Portal with specific location and run Set-AzSKSubscriptionSecurity  command against that sub.

**Comment by cmenzi** [comment]
Yes, that works for the ResourceGroup, but not storagesccount that is created inside ResourceGroup. I suggest that all resources inside the RG should take over its location. 

Having a OrgPolicy is not I like to setup for now.
